metricbeat/_meta/beat.full.yml

```diff
@@ -117,7 +117,7 @@ metricbeat.modules:
 
 #------------------------------- MongoDB Module ------------------------------
 #- module: mongodb
-  #metricsets: ["status"]
+  #metricsets: ["status", "dbstats"]
```

Comment:
We should probably sort this alphabetically. But not sure if we are consistent in other modules with that ...